lib/pod/command/package.rb

```diff
@@ -67,10 +67,13 @@ def build_in_sandbox(platform)
 
         sandbox = install_pod(platform.name)
 
-        perform_build(platform, sandbox)
```

Comment:
Correct me if I'm wrong here, but don't we need to wrap lines 74-75 in an `ensure` block so that they're cleaned up even if the build fails?